src/components/SoundBox/SoundBox.test.js

```diff
@@ -15,6 +15,11 @@ describe('SoundBox', () => {
     expect(renderedSoundBox.find(SandButton)).toHaveLength(4);
     expect(renderedSoundBox.find(VioletButton)).toHaveLength(4);
   });
+  it('renders SoundBox if browser uses webkitAudioContext', () => {
+    window.AudioContext = window.AudioContext || window.webkitAudioContext;
```

Comment:
Please tell me the purpose of this code? I delete this line and the test will be green

Reply:
I fixed this in next commit